src/Microsoft.DocAsCode.Metadata.ManagedReference/Microsoft.DocAsCode.Metadata.ManagedReference.csproj

```diff
@@ -129,6 +130,9 @@
       <Name>Microsoft.DocAsCode.YamlSerialization</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Transform\TripleSlashCommentTransform.xsl" />
```

Comment:
You may rename the resource as what docfx.csproj does. The behavior for Resource Name in project.json is different from the one in csproj

---
In reply to: [59156174](https://github.com/dotnet/docfx/pull/249#discussion_r59156174) [](ancestors = 59156174)